applications/nrf_desktop/README.rst

```diff
@@ -8,15 +8,25 @@ Nordic Semiconductor HID reference design.
 Overview
 ********
 
-This application is a reference design of a HID device.
-
 The code is currently work-in-progress and as such is not fully functional, verified, or
 supported for product development.
 
+Welcome to the firmware documentation for nRF52 Desktop, a reference design of
```

Comment:
No need to welcome, I'd just state what nRF52 Desktop is.

Reply:
done